src/js/providers/html5.ts

```diff
@@ -179,11 +179,18 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         },
 
         canplay(this: ProviderWithMixins): void {
-            _canSeek = true;
-            if (!_androidHls) {
-                _setMediaType();
+            if (OS.iOS && Browser.safari) {
+                return;
```

Comment:
The key to this issue is exiting out of `canplay` but we only need to do this if we're seeking and we've already handled `canplay` for the current media.